README.md

```diff
@@ -285,9 +321,31 @@ visible | Whether the element is visible to the user | 'true'
 view-tag | The tag value assigned to the element. Could be `null` | 'my tag'
 
 
-## Element Location
+## Compose Element Attributes
+
+Espresso driver supports the following element attributes in `compose` subdriver:
+
+Name | Description | Example
+--- | --- | ---
+bounds | The element's visible frame (`[left, top][right, bottom]`) | `[0,0][100,100]`
+checked | Whether the element is checked. Always `false` if the element is not checkable | 'false'
+class | The full name of the element's class. Could be `ComposeNode` for some elements | 'ComposeNode'
+clickable | Whether the element could be clicked | 'false'
+content-desc | The content-description attribute of the accessible element | 'foo'
+enabled | Whether the element could be clicked | 'true'
+focused | Whether the element could is focused. Always `false` if the element is not focusable | 'false'
+index | Element's index in the tree hierarchy | `0`
+password | Whether the element is a password input field | 'true'
+resource-id | Element's resource identifier. Could be `null` | 'com.mycompany:id/resId'
+scrollable | Whether the element is scrollable | 'true'
+selected | Whether the element is selected | 'false'
+text | The element's text | 'my text'
+view-tag | The tag value assigned to the element. Could be `null` | 'my tag'
```

Comment:
can be changed to => The [testTag](https://developer.android.com/reference/kotlin/androidx/compose/ui/platform/package-summary#(androidx.compose.ui.Modifier).testTag(kotlin.String)) value assigned to the element

Reply:
changed